spec/02-integration/05-proxy/04-plugins_triggering_spec.lua

```diff
@@ -16,7 +16,7 @@ for _, strategy in helpers.each_strategy() do
     local db
     local bp
 
-    lazy_setup(function()
+    setup(function()
```

Comment:
is this change necessary for proper operation or was it temporary debugging?

Reply:
oh, nice catch.